modules/core/src/test/scala/JsonExamples.scala

```diff
@@ -0,0 +1,133 @@
+package scalaz
+
+package schema
+
+import scalaz.Scalaz._
+import testz._
+
+object JsonExamples {
+
+  def tests[T](harness: Harness[T]): T = {
+    import harness._
+    import scalaz.schema.Json._
+    import scalaz.schema.Json.module._
+
+    def matchJsonStrings(a: String, b: String): Boolean =
+      a.toLowerCase.replaceAll("\\s+", "") == b.toLowerCase.replaceAll("\\s+", "")
+
+    section("JSON Schema Tests")(
+      test("Case Class should Serialize using Schema") { () =>
```

Comment:
Can you also add one test for case when `Union` misses one branch?

Reply:
done! It should be removed once it becomes impossible by construction.